dev/com.ibm.ws.jsf.2.3_fat/semeruFips140_3CustomProfile.properties

```diff
@@ -8,7 +8,6 @@ RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.desc.name = Custom OpenJCEPl
 RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.extends = RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3
 RestrictedSecurity.OpenJCEPlusFIPS.FIPS140-3-Custom.jce.provider.2 = sun.security.provider.Sun [+ \
     {KeyStore, PKCS12, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
-    {KeyStore, JKS, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
     {MessageDigest, SHA-1, *, ModuleAndFullClassName:java.base/java.security.KeyStore}, \
```

Comment:
we should remove the SHA-1 for java.security.KeyStore here as well.

Reply:
if we didn't convert the jks to p12, wouldn't that mean we need to keep line 11 as well?

however, i see that the bucket is passing on the latest fips build, so i'll approve for now.